src/components/hmrc-header/_header.scss

```diff
@@ -0,0 +1,298 @@
+@import "../../settings/all";
```

Comment:
all these styles should be coming from govuk-frontend so i don't think we need this file